third_party/terraform/website/docs/d/compute_image.html.markdown

```diff
@@ -35,10 +35,11 @@ resource "google_compute_instance" "default" {
 
 The following arguments are supported:
 
-* `name` or `family` - (Required) The name of a specific image or a family.
+* `name`, `family` or `filter` - (Required) The name of a specific image or a family.
 Exactly one of `name` of `family` must be specified. If `name` is specified, it will fetch
 the corresponding image. If `family` is specified, it will returns the latest image
```

Comment:
Would you mind correcting the `it will returns` to `it will return` since you're in here already :)

Reply:
fixed